sys/env.go

```diff
@@ -20,14 +20,14 @@ package sys
 
 import "golang.org/x/sys/unix"
 
-// RunningPrivileged returns true if the effective user ID of the
+// runningPrivileged returns true if the effective user ID of the
 // calling process is 0
-func RunningPrivileged() bool {
+func runningPrivileged() bool {
 	return unix.Geteuid() == 0
 }
```

Comment:
Do we even need these functions? We could just use os.Geteuid directly.

Reply:
Yes, that would be fine with me as well; at least I think we can drop one of the two (`!runningPrivileged()` is probably just as readable); but let me have a quick peek again to see what that would look like (otherwise we can move it into one of tthe `oom`  files)